packages/notification-center-vue/src/index.ts

```diff
@@ -1 +1,26 @@
-export * from './lib';
+import type { App } from 'vue';
+import FloatingVue from 'floating-vue';
+import 'floating-vue/dist/style.css';
+
+import { NotificationCenterComponent } from './lib';
+
+export { NotificationCenterComponent } from './lib';
+
+export default {
+  install(app: App) {
+    app.use(FloatingVue, {
+      themes: {
+        dark: {
+          $extend: 'dropdown',
+          distance: 10,
+        },
+        light: {
+          $extend: 'dropdown',
+          distance: 10,
+        },
+      },
+    });
+
+    app.component('NotificationCenterComponent', NotificationCenterComponent);
+  },
+};
```

Comment:
This is a plugin I told you about before. 
It installs the popover library and configures it with some defaults, and creates the light/dark themes that we use later.
Also, it adds the `NotificationCenterComponent` to the app, so it's not needed to import it in the Vue file.